buildable/build.gradle

```diff
@@ -14,63 +14,70 @@ artifacts {
     archives sourcesJar
 }
 
-signing {
-    sign configurations.archives
+dependencies{
+    compile group: 'com.squareup', name:'javapoet', version: '1.7.0'
+    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.4'
```

Comment:
If I'm correct and we're only pulling it in for those two small methods, let's not include commons-lang3.